testdata/ast/golden/3.4/binop3.out

```diff
@@ -0,0 +1,8 @@
+(-1, -1)     Module               	prefix=||	suffix=||	indent=||
+(1, 0)       Expr                 	prefix=||	suffix=||	indent=||
+(1, 0)       BinOp                	prefix=||	suffix=||	indent=||
+(1, 0)       Name a               	prefix=||	suffix=| |	indent=||
+(-1, -1)     MatMult              	prefix=||	suffix=||	indent=||
```

Comment:
This doesn't look right to me. AFAICT python 3.4 doesn't have `ast.MatMult`. How did you generate this?

Reply:
Oh. I blindly copied it, I'll delete it. 

Maybe the test should complain if an out file exists if the test isn't being run.